src/WebJobs.Script.WebHost/Metrics/LinuxContainerMetricsPublisher.cs

```diff
@@ -86,12 +88,19 @@ public LinuxContainerMetricsPublisher(IEnvironment environment, IOptionsMonitor<
             _httpClient = (httpClient != null) ? httpClient : CreateMetricsPublisherHttpClient();
             if (_standbyOptions.CurrentValue.InStandbyMode)
             {
-                _standbyOptionsOnChangeSubscription = _standbyOptions.OnChange(o => OnStandbyOptionsChange());
+                _standbyOptionsOnChangeListener = _standbyOptions.OnChange(o => OnStandbyOptionsChange());
             }
             else
             {
                 Start();
             }
+
+            _hostingConfigOptionsOnChangeListener = _hostingConfigOptions.OnChange(OnHostingConfigOptionsChanged);
+        }
+
+        private void OnHostingConfigOptionsChanged(FunctionsHostingConfigOptions newOptions)
+        {
+            _logger.LogInformation("CGroup memory metrics enabled: {Enabled}", newOptions.IsCGroupMemoryMetricsEnabled);
```

Comment:
Does this need to be logged at Information level? Not sure if customers need this info. And should we log every time the config changes even if some other property (and not `IsCsGroupMemoryMetricsEnabled`) is changed?

Reply:
No. I pushed an update so that it gets logged if the value of the property changes.

Reply:
This logger will not log to customer logs.